.github/workflows/arm64.yml

```diff
@@ -67,6 +67,14 @@ jobs:
           ${TAS_SCRIPTS}/wait-for-alfresco-start.sh "http://localhost:8080/alfresco/" 180
           docker ps
           curl -v --user admin.pipeline@alfresco.com:admin http://localhost:8080/alfresco/api/discovery
+      - name: "Persist environment setup failure logs"
```

Comment:
I would rename this to _"Echo ..."_ or _"Print ..."_ rather than _"Persist ..."_. Makes it clearer IMHO.